### DISCUSSION TITLE
Paid Courses

### DISCUSSION THREAD

**Discussion by IrrationalBoolean** [dailydotdev/daily#752] [Feature Requests]
Paid Courses should be clearly labeled as such.  The "courses" tag should be broken up into free/paid.

**Comment by idoshamun** [comment]
Hi @IrrationalBoolean,

Thanks for the suggestion. I'd love if you could share some examples that you found on daily.dev for this case.

**Comment by IrrationalBoolean** [comment]
https://app.daily.dev/posts/l-lpW8Yln : https://laravel-news.com/unlock-the-power-of-tdd

Here's the daily.dev link and the site link to the one that prompted me to put in this discussion.

The page itself offers no value unless you pay.  Mostly just an ad for the course.

**Reply by idoshamun** [reply]
Thanks for sharing the examples. We'll check what we can do about it
